src/main/java/org/terasology/structureTemplates/util/BlockRegionTransform.java

```diff
@@ -92,7 +92,7 @@ public Region3i transformRegion(Region3i region) {
 
 
     public BlockRegion transformRegion(BlockRegion region) {
-        return new BlockRegion(transformVector3i(region.getMin(new Vector3i())),
+        return new BlockRegion().union(transformVector3i(region.getMin(new Vector3i()))).union(
```

Comment:
Isn't `correctBounds` doing exactly what we need here?
```suggestion
        return new BlockRegion(
        	transformVector3i(region.getMin(new Vector3i())),
        	transformVector3i(region.getMax(new Vector3i()))).correctBounds();

```

Reply:
that is also valid. Its just a matter of choice at this point. I guess correctBounds might be more faster but Its more of a matter of choice. you think correctBounds makes more sense in this case?

Reply:
I'd say it makes more sense, yes. Reason being that it's better understandable in what's going on. I mean if the bounds are incorrect, why not use a descriptive call to "correctBounds" ^^